packages/xo-web/src/xo-app/backup-ng/new/utils.js

```diff
@@ -38,3 +39,35 @@ Number.propTypes = {
   onChange: PropTypes.func.isRequired,
   value: PropTypes.number.isRequired,
 }
+
+export const FormFeedback = ({
+  showError,
+  error,
+  component: Component,
+  ...props
+}) => (
+  <div>
+    <Component
+      {...props}
+      style={
+        showError === undefined
+          ? undefined
+          : {
+              borderColor: showError ? 'red' : 'green',
```

Comment:
What if the component doesn't have a border?

Reply:
This component is specific to the current Backup NG UI.